governance/polls/Proposal - Add PAX as a Collateral Type - August 24, 2020.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Add PAX as a Collateral Type - August 24, 2020
+summary: This poll proposes Risk Parameters which could be used to initialize PAX as a new asset class.
+discussion_link: https://forum.makerdao.com/t/usdt-pax-mip12c2-sp2-collateral-onboarding-risk-evaluation/3723
+vote_type: Plurality Voting
+options:
+   0: Abstain
+   1: Yes
+   2: No
+---
+# Poll: Add PAX as a Collateral Type - August 24, 2020
+
+The Maker Foundation Interim Risk Team has placed a Governance Poll into the [voting system](https://vote.makerdao.com/polling) through which the community can signal their support to add PAX (Paxos) as a collateral type.
```

Comment:
I think technically this should be the MakerDAO Risk Team at this point, but will defer to you.